### DISCUSSION TITLE
Welcome to NoteBook Discussions!

### DISCUSSION THREAD

**Discussion by Rajat2024** [Rajat2024/NoteBook#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chaitanyarora** [comment]
What is the significance of SECRET_KEY ?

**Reply by kumar007ambi** [reply]
if your database has authentication, then what password you have set provide as secret key.
Mostly root is used for the password

**Reply by Rajat2024** [reply]
@chaitanyarora 
Use of secret key is to perform operation on particular user
Here it is use jwt token

**Comment by elhussienalmasri** [comment]
hey @Rajat2024 , I create an issue from  more than two days ,
and ask you to assign this issue for me , and no one do, can you please tell me what must
I do ? 
can I make PR or wait to assign the issue for me

**Reply by Rajat2024** [reply]
@elhussienalmasri  wait for 2 more days my sem exam is going till 26 may
